rules/list-item.js

```diff
@@ -276,7 +276,7 @@ function validateListItemPrefix(descriptionText, prefixText) {
 }
 
 function validateListItemSuffix(descriptionText, suffixText) {
-	if (/[.!?]\s*$/.test(suffixText)) {
+	if (/[.!?…]\s*$/.test(suffixText)) {
 		// Description ends with '.', '!', or '?'
```

Comment:
You need to update this comment too.

Reply:
And probably https://github.com/sindresorhus/awesome-lint/pull/98/files#diff-6e9aa7b10a8503bfb11fbb683cfe854bR284 too.